src/api/app/helpers/watchlist_helper.rb

```diff
@@ -0,0 +1,15 @@
+module WatchlistHelper
+  def self.render(my_model, xml)
```

Comment:
Hm why is this a helper and not a view?

Reply:
Because this code is used twice and we don't have a clear way to create partials using `XML builder`. That's why I moved it to a helper to keep it DRY and we can not simply render this XML in another root element, XML builder doesn't allow that.

Reply:
It should be possible to render a partial in a view relying on XML builder. This is much easier to understand than having a helper which is acting as a partial.

Reply:
Thank you. It's updated